### PR TITLE
docs(website): reduce Contributor_Tiers page's table vertical Padding

### LINKED ISSUE #8805: Docs: Unusually large padding after table cell lines in Contributor_Tiers.mdx
### Before You File a Documentation Request Please Confirm You Have Done The Following...

- [X] I have looked for existing [open or closed documentation requests](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aissue+label%3Adocumentation) that match my proposal.
- [X] I have [read the FAQ](https://typescript-eslint.io/linting/troubleshooting) and my problem is not listed.

### Suggested Changes

https://typescript-eslint.io/maintenance/contributor-tiers includes a few table cells that have multiple lines of content. For example, _Entry Requirements_ > _Committer_. There are `<br />s` between each `<p>` in those cells to make separate their content across lines.

Right now, there's something like twice as much vertical padding after each of those lines compared to what we'd expect:

Screenshot of the Contributor Tiers table with 2-3 line-heights of padding between cell lines in Committer and Maintainer

Proposal: let's tweak the styles for those tables to not have so much vertical padding?

### Affected URL(s)

https://typescript-eslint.io/maintenance/contributor-tiers/

### PR DESCRIPTION
**Pull request by developer-bandi**
## PR Checklist

- [x] Addresses an existing open issue: fixes #8805
- [x] That issue was marked as [accepting prs](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aopen+is%3Aissue+label%3A%22accepting+prs%22)
- [x] Steps in [Contributing](https://typescript-eslint.io/contributing) were taken

## Overview

by replace br to new line, vertical padding is reduced

### CODE REVIEW

File: docs/maintenance/Contributor_Tiers.mdx [hunk 1]
```diff
@@ -52,15 +52,16 @@ These tiers of contributors, committers, and maintainers roughly reflect how we'
...
         ~10 points in pull requests
-        <br />
+
         ~30 points total
         <br />3 of 6 active months
       </td>
       <td>~20 points</td>
       <td>~2 shares / active month</td>
     </tr>
+
```
Comment (JoshuaKGoldberg) [inline docs/maintenance/Contributor_Tiers.mdx:64]: Nit: unrelated?

```suggestion
```

File: docs/maintenance/Contributor_Tiers.mdx
Comment (JoshuaKGoldberg) [inline docs/maintenance/Contributor_Tiers.mdx:1]: This is most of the way there, but there's still some padding below the paragraphs that's a little awkward:

Screenshot of the updated table with some padding below each multiline paragraph still

I think you'll have to change some CSS around to get rid of it.

**Review by JoshuaKGoldberg — CHANGES_REQUESTED** [review]
Getting there! 💪

File: docs/maintenance/Contributor_Tiers.mdx
Comment (developer-bandi) [reply docs/maintenance/Contributor_Tiers.mdx:1]: i fix margin like below
스크린샷 2024-06-02 오후 7 58 49

File: docs/maintenance/Contributor_Tiers.mdx (hunk 1 above)
Comment (developer-bandi) [reply docs/maintenance/Contributor_Tiers.mdx:64]: this space is evoke with format script. and i fix this, the format ci is failed.

**Review by JoshuaKGoldberg — APPROVED** [review]
Super, thanks!
